guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,455 @@
+
+The fabric8 Maven plugin will place all the jar files under `/maven` directory which need to be added to the JVM's classpath. Also note that we have added the Akka clustering related system properties as part of the JVM options.
+
+Replace `com.mycompany.MainClass` with the actual main class of your application.
+
+Execute the following command to containerize and publish your application's Docker image.
+
+```bash
+mvn clean package docker:build
+```
+
+### Creating Kubernetes Service for Akka remoting
+
+Next we will declare the Akka remoting port we'd like to expose from our Pod using Kubernetes Service so it can be referenced from one Pod to another.
+
+Execute the following command to create Kubernetes Service for the application's Akka remoting port.
+
+```bash
+cat << EOF | kubectl create -f -
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "app": "myapp"
+    },
+    "name": "myapp-akka-remoting"
+  },
+  "spec": {
+    "clusterIP": "None",
+    "ports": [
+      {
+        "port": 2551,
+        "protocol": "TCP",
+        "targetPort": 2551
+      }
+    ],
+    "selector": {
+      "app": "myapp"
+    }
+  }
+}
+EOF
+```
+
+This Kubernetes Service has `myapp-akka-remoting` as its name. The service exposes TCP port `2551` which is the application's Akka remoting port.
+
+The service will be applied to the Pods based on the `selector` value. All Pods which has a label called `app` with the value of `myapp` will have the TCP port `2551` exposed.
+
+View the list of Kubernetes Service using the following command.
+
+```bash
+kubectl get services
+```
+
+The service has a label called `app` with the value of `myapp`, which will allow filtering list of Service based on this label.
+
+```bash
+kubectl get services --selector=app=myapp
+```
+
+### Creating Kubernetes StatefulSet resource
+
+Once we have the application containerized and published, and the Kubernetes Service declared for the application's Akka remoting port, we are ready to deploy the application.
+
+Execute the following command to deploy the application.
+
+```bash
+cat << EOF | kubectl create -f -
+{
+  "apiVersion": "apps/v1beta1",
+  "kind": "StatefulSet",
+  "metadata": {
+    "name": "myapp"
+  },
+  "spec": {
+    "serviceName": "myapp",
+    "replicas": 1,
+    "template": {
+      "metadata": {
+        "labels": {
+          "app": "myapp"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "myapp",
+            "image": "mygroup/myapp",
+            "imagePullPolicy": "Never",
+            "ports": [
+              {
+                "containerPort": 2551,
+                "name": "akka-remote"
+              }
+            ],
+            "resources": {
```

Comment:
After thinking about this, I'm actually inclined to keep it since we have this declared in our `bundle.conf` as well. I also mentioned that the user should adjust these resource values according to their app.